doc/modules/linear_model.rst

```diff
@@ -266,6 +266,11 @@ They also tend to break when the problem is badly conditioned
 
   * :ref:`sphx_glr_auto_examples_linear_model_plot_lasso_model_selection.py`
 
+.. note:: **Comparison with the regularization parameter of SVM**
```

Comment:
wouldn't it be better to mention that directly after alpha is introduces? Also, there are only two possiblities, right? alpha = 1/C or alpha = n_samples / C?